Documentation (#78)
ropensci-review-tools/babelquarto: README.md

pak::pak("ropensci-review-tools/babelquarto")
```

## Getting Started

> The {babelquarto} package allows you to create a multilingual Quarto project (a book or a website).

Reviewer: ```suggestion
The {babelquarto} package allows you to create and render a multilingual Quarto project, book or website.
```

I try to avoid parentheses as I feel they break the flow somehow.

Author: Noted 😉

A multilingual project is based on a main language and can feature any number of additional languages.
The languages are registered once and are then present in your `_quarto.yml` configuration file under the `babelquarto` key.

> Each Quarto markdown file in your project can then be translated into these further languages and these will be used to generate the project in each language.

Reviewer: ```suggestion
Each Quarto Markdown file in your project can then be translated into these further languages and these will be used to generate the project in each language.
```

If you start from scratch, you might want to look at `babelquarto::quarto_multilingual_book()` or `babelquarto::quarto_multilingual_website()` and read `vignette("babelquarto")`.

If you already have and existing Quarto project and want to convert it to a multilingual project, you can use `babelquarto::register_main_language()` and `babelquarto::register_further_languages()` to get started. For more information you can read `vignette("convert)`.

## Examples

To get a feel of what a multilingual book can look like, you can have a look at this book: [*rOpenSci Packages: Development, Maintenance, and Peer Review*](https://devguide.ropensci.org/).

> For a multilingual website, you can check out [@joelnitta's website](https://www.joelnitta.com/).

Reviewer: ```suggestion
For a multilingual website, you can check out [Joel Nitta's website](https://www.joelnitta.com/).
```

pak::pak("ropensci-review-tools/babelquarto")
```

> ## Getting Started

Reviewer: and maybe also links to examples. Basically more of a showcase.

> ## Getting Started

Author: I've made the Getting started part a bit more detailed.

For the links, do you have any ideas? The ropensci devguide obviously. But any other projects you are aware of that we could mention?

> ## Getting Started

Reviewer: @joelnitta's website https://github.com/joelnitta/joelnitta-home

> ## Getting Started

Author: 👍
